slackrelay.py

```diff
@@ -414,6 +414,7 @@ def main():
       # Pass event to the backend for each matching rule
       for r in matchingRules:
         logging.debug("Processing rule: %s" % r.name)
+        req = False
```

Comment:
agreed, but ISTM on balance it is better to leave it in (conservative assignment making it likelier we won't get false confirmations due to overlooked logic flaws)